src/test/java/com/parkingapp/parkingservice/infrastructure/entrypoint/rest/VehiclesControllerContractTest.java

```diff
@@ -181,4 +183,36 @@ private MockMvcResponse whenARequestToCreateAVehicleIsReceived(String requestBod
                     .post("/vehicles");
         }
     }
+
+    @Nested
+    class GetUserVehicles {
+        @Test
+        void shouldGetUserVehicles() throws Exception {
+            // GIVEN
+            List<Vehicle> vehicleList =  List.of(newVehicle);
+            when(idGenerator.generate()).thenReturn(userId);
```

Comment:
```suggestion

```

The IdGenerator is not invoked in this method

Reply:
Removed!